calculator/operations.py

```diff
@@ -1,23 +1,23 @@
+from functools import reduce
+import operator as op
+from sympy import symbols
+from sympy.plotting import plot
+from sympy.parsing.sympy_parser import parse_expr
 
 def add(*args):
-    # your implementation here
-    pass
+    return sum(args)
 
 def subtract(*args):
-    # your implementation here
-    pass
+    return reduce(op.sub, args)
```

Comment:
Thanks, and I reiterate: Why do the work when you can have something else do the work for you? ;)